src/modules/commander/accelerometer_calibration.cpp

```diff
@@ -175,7 +175,7 @@ typedef struct  {
 
 int do_accel_calibration(orb_advert_t *mavlink_log_pub)
 {
-#ifndef __PX4_QURT
+#if !defined(__PX4_QURT) && !defined(__PX4_POSIX_EAGLE)
```

Comment:
Is there a specific reason why you use commander on the Linux side? Or do you just want to have the possibility to run it on either?